mne/channels/channels.py

```diff
@@ -385,7 +385,7 @@ def set_channel_types(self, mapping, *, on_unit_change="warn", verbose=None):
         mapping : dict
             A dictionary mapping channel names to sensor types, e.g.,
             ``{'EEG061': 'eog'}``.
-        on_unit_change : 'raise' | 'warn' | 'ignore'
+        on_unit_change : ``'raise'`` | ``'warn'`` | ``'ignore'``
```

Comment:
I don't think we've decided and I looked at numpydoc but it doesn't seem to say

https://numpydoc.readthedocs.io/en/latest/format.html#parameters

In the meantime I think this should render more nicely. In principle maybe numpydoc should autodetect such strings and literal-ify them but I wonder if it would befragile. So maybe we should turn them to literals for now and someday if someone wants to make numpydoc smarter they can, and it's easy enough to replace our instances at that point.

Reply:
Good point. I've definitely seen it elsewhere in the codebase, and IMO it renders better. But actually in numpydoc:

```
When a parameter can only assume one of a fixed set of values, 
those values can be listed in braces, with the default appearing first:

order : {'C', 'F', 'A'}
    Description of `order`.
```

Not sure how it renders